Content.Server/GameObjects/Components/Cargo/CargoConsoleComponent.cs

```diff
@@ -53,8 +75,9 @@ public override void Initialize()
             _userInterface = Owner.GetComponent<ServerUserInterfaceComponent>().GetBoundUserInterface(CargoConsoleUiKey.Key);
             _userInterface.OnReceiveMessage += UserInterfaceOnOnReceiveMessage;
             _powerDevice = Owner.GetComponent<PowerDeviceComponent>();
-            _galacticBankManager.AddComponent(this);
-            BankId = 0;
+            EntitySystem.TryGet(out _cargoConsoleSystem);
```

Comment:
Any reason for using `TryGet` over `Get` here?

Reply:
The reason is that I didn't know that existed, I'll change it now